Build Error in Java 11

I am attempting to build a local checkout of this project, but I get following error: 

```
➜ ./gradlew build 

Welcome to Gradle 5.6.3!

Here are the highlights of this release:
 - Incremental Groovy compilation
 - Groovy compile avoidance
 - Test fixtures for Java projects
 - Manage plugin versions via settings script

For more details see https://docs.gradle.org/5.6.3/release-notes.html

Starting a Gradle Daemon, 1 incompatible Daemon could not be reused, use --status for details
Configuration(s) specified but the install task does not exist in project :.
Configuration(s) specified but the install task does not exist in project :kgraphql.
Configuration(s) specified but the install task does not exist in project :kgraphql-ktor.

> Task :kgraphql:compileKotlin FAILED
e: .../kgraphql/src/main/kotlin/com/apurebase/kgraphql/request/Lexer.kt: (8, 37): Symbol is declared in module 'jdk.scripting.nashorn' which does not export package 'jdk.nashorn.internal.r
untime'
e: .../KGraphQL/kgraphql/src/main/kotlin/com/apurebase/kgraphql/request/Lexer.kt: (528, 26): Symbol is declared in module 'jdk.scripting.nashorn' which does not export package 'jdk.nashorn.internal
.runtime'
e: .../KGraphQL/kgraphql/src/main/kotlin/com/apurebase/kgraphql/request/Lexer.kt: (528, 40): Symbol is declared in module 'jdk.scripting.nashorn' which does not export package 'jdk.nashorn.internal
.runtime'

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':kgraphql:compileKotlin'.
> Compilation error. See log for more details

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 11s
4 actionable tasks: 1 executed, 3 up-to-date
```

```
➜ java -version      
openjdk version "11.0.5" 2019-10-15 LTS
OpenJDK Runtime Environment Corretto-11.0.5.10.1 (build 11.0.5+10-LTS)
OpenJDK 64-Bit Server VM Corretto-11.0.5.10.1 (build 11.0.5+10-LTS, mixed mode)
```

Can you please advise how to fix this ? 

It seems Lexer.kt uses some internal API that is not exposed.

--- a/kgraphql/src/main/kotlin/com/apurebase/kgraphql/request/Lexer.kt
+++ b/kgraphql/src/main/kotlin/com/apurebase/kgraphql/request/Lexer.kt
@@ -5,7 +5,6 @@ import com.apurebase.kgraphql.schema.model.ast.Token
 import com.apurebase.kgraphql.schema.model.ast.TokenKindEnum
 import com.apurebase.kgraphql.schema.model.ast.TokenKindEnum.*
 import com.apurebase.kgraphql.schema.structure.dedentBlockStringValue
-import jdk.nashorn.internal.runtime.JSONFunctions
 
 data class Lexer(
     val source: Source,
@@ -522,13 +521,32 @@ data class Lexer(
         else -> -1
     }
 
-    private fun printCharCode(code: Int?) = when {
-        code == null -> EOF.str
-        // Trust JSON for ASCII.
-        code < 0x007f -> JSONFunctions.quote(code.toChar().toString())
-        // Otherwise print the escaped form.
-        else -> "\"\\u${("00" + code.toString(16).toUpperCase()).takeLast(4)}\""
-    }
+    private fun printCharCode(code: Int?) =
+        if (code == null) EOF.str
+        else StringBuilder().apply {
+            append("\"")
+            when (val char = code.toChar()) {
+                '\\' -> append("\\\\")
+                '"' -> append("\\\"")
+                '\b' -> append("\\b")
+                '\n' -> append("\\n")
+                '\r' -> append("\\r")
+                '\t' -> append("\\t")
+                else ->
+                    if (code < 0x007f && char > ' ') {
+                        // Refer: https://utf8-chartable.de/unicode-utf8-table.pl?number=128
+                        append(char)
+                    } else {
+                        append("\\u")
+                        val hex = Integer.toHexString(code)
+                        for (i in hex.length..3) {
+                            append('0')
+                        }
+                        append(hex.toUpperCase())
+                    }
+            }
+            append("\"")
+        }.toString()
 
     /**
      * Report a message that an unexpected character was encountered.

